### DISCUSSION TITLE
Walletkit with Docker-compose

### DISCUSSION THREAD

**Discussion by vanhoanHoang** [walt-id/.github#49] [Help] [answered]
Hi team,

I got some problems when running docker-compose of walletkit on Windows with the latest versions of all images: issuer-portal, wallet-portal, verifier-portal, and wallet-kit. Even though, all components launched successfully, when I click on the VC request in the Wallet portal, I always get the 400 error from Issuer.

However, when I set all these images version to v0.6.0, all the components run successfully without any problem.
Does anyone get the same problem with the latest version and have a workaround please.

Thank you

**Comment by severinstampler** [comment, marked as answer]
Hi @vanhoanHoang 
I committed a fix in the docker-compose configuration files.
I hope this fixes your problem too. Let me knowif it works.
Thanks.

**Comment by vanhoanHoang** [comment]
@severinstampler  thank you very much. It worked.
